ConcernsCaseWork/ConcernsCaseWork.Data/Gateways/TrustFinancialForecastGateway.cs

```diff
@@ -35,4 +35,16 @@ public async Task<int> Update(TrustFinancialForecast trustFinancialForecast, Can
 		
 		return trustFinancialForecast.Id;
 	}
+
+	public void Delete(int trustFinancialForecastId)
+	{
+		var result = this._concernsDbContext.TrustFinancialForecasts.SingleOrDefault(f => f.Id == trustFinancialForecastId);
+		if (result != null)
```

Comment:
I don't think we need this check because a previous get would have guarded against it?

Reply:
Remove null check